src/hatti/ona/forms.cljs

```diff
@@ -239,13 +264,27 @@
     (or (image? field)
         (video? field)) (let [image (:download_url answer)
                               thumb (or (:small_download_url answer) image)
+                              media-upload-fail-hlp-url
```

Comment:
add an `e` to `media-upload-fail-hlp-url` since it's already a long variable and it will be more descriptive